setup.py

```diff
@@ -68,15 +68,16 @@ def get_all_extras() -> Dict:
 
     cli_deps = [
         "click",
-        "PyYAML",
-        "jsonschema",
+        "pyyaml>=4.2b1",
```

Comment:
we should mirror Pipfile[packages] here imo

Reply:
not applicable anymore